webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+const ProgressBarPlugin = require('progress-bar-webpack-plugin');
+
+module.exports = {
+  resolve: {
+    extensions: ['.js', '.jsx', '.ts', '.tsx', '.json']
+  },
+  plugins: [
+    new ProgressBarPlugin()
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.[jt]sx?$/,
+        include: [
+          path.resolve(__dirname, 'src'),
+          path.resolve(__dirname, 'styleguide')
+        ],
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              presets: [['buildo', { env: 'react' }]]
+            }
+          },
+          {
+            loader: 'ts-loader',
+            options: {
+              transpileOnly: true,
```

Comment:
this + the fact that there's no typescript build run on the CI makes TS quite useless..

Reply:
you're right, we should add a script `typecheck` that runs `tsc --noEmit` and call it in the CI!